receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -55,6 +55,13 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [database]
+  postgresql.database.count:
+    attributes: []
+    description: Number of user databases.
+    enabled: true
+    gauge:
```

Comment:
Arguably, this should be a non-monotonic counter because it can be meaningfully summed. For example, a user may wish to know the total number of databases across their environment.